### ISSUE TITLE
Label "for" property not populated

### ISSUE THREAD

**Issue by iwllyu** [zabute/formsy-semantic-ui-react#16]
when you provide a `name` prop in semantic-ui-react input, the "for" label is populated in the rendered HTML.

Semantic UI React
```
<Form.Input
  id="name"
  name="name"
  value={this.state.name}
  icon="shopping bag"
  iconPosition="left"
  onChange={this.handleChange}
  width
/>
```

Rendered HTML
```
<label for="name">Name</label>
<div class="field">
  <div class="ui left icon input">
    <i aria-hidden="true" class="shopping bag icon"></i>
    <input type="text" id="name" name="name" value="">
  </div>
</div>
```

formsy-semantic-ui-react
```
<Form.Input
  required
  name="firstName"
  label="First name"
  placeholder="First name"
  validations="isWords"
  errorLabel={ <Label color="red" pointing/> }
  validationErrors={{
    isWords: 'No numbers or special characters allowed',
    isDefaultRequiredValue: 'Fist Name is Required',
  }}
/>
```

rendered html
```
<label><!-- react-text: 92 --> <!-- /react-text --><!-- react-text: 93 -->First name<!-- /react-text --><!-- react-text: 94 --> <!-- /react-text --></label>
<div class="ui input">
  <input type="text" required="" name="firstName" placeholder="First name" value="">
</div>
```

 This enables the input to be focused when you click the label.

**Comment by iwllyu** [comment]
nevermind: this is an issue with semantic ui react itself

**Comment by iwllyu** [comment]
nevermind it's actually not a problem at all!

**Comment by iwllyu** [comment]
Sorry about being all over the place with this. So SUI has it implemented properly, where if you provide an ID to Form.Input it will do the focus behavior

https://codepen.io/anon/pen/yoPJQd?editors=0010

but formsy-SUI-R does not

**Comment by noinkling** [comment]
Can confirm.

**Comment by zabute** [comment]
fixed in ```0.2.10```
